freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -184,6 +192,15 @@ public void cleanupByCluster(VaultCleanupRequest request, String accountId) thro
         }
     }
 
+    private void cleanupSecrets(String environmentCrn, String clusterCrn, String accountId) throws DeleteException {
+        VaultPathBuilder b = new VaultPathBuilder()
```

Comment:
variable should be renamed to something more informative, like `vaultPathBuilder`

Reply:
Done